AutoGear.lua

```diff
@@ -4184,7 +4188,7 @@ function AutoGearMain()
 									table.remove(futureAction, i)
 								end
 							else
-								PickupContainerItem(curAction.container, curAction.slot)
+								((C_Container and C_Container.PickupContainerItem) or PickupContainerItem)(curAction.container, curAction.slot)
```

Comment:
I'm curious how this worked while the comment in a change above for this exact same function claims to not have worked.

Reply:
You and me both, it was somewhat frustrating that the above one that didn't work was the only one. It kept throwing an error

Reply:
BugGrabber often groups identical errors together, even if they occur in different parts of the code.  It's possible both parts were throwing the error, but the stack trace only showed the first one, if you were looking at BugGrabber.  Alternatively, it's also possible that your testing only hit the other code block, so this part was not executed.

Reply:
I am using BugGrabber and I was thinking the same, possibly I only hit the one code block that was erroring. I am planning to dig into that a little more.

Reply:
I see you've now updated the other instance of this code, but not this instance.  If the other instance needed to be updated because it caused an error, it's very likely this one needs to be updated as well.